src/service/models.py

```diff
@@ -13,71 +13,3 @@ class ErrorResponse(BaseModel):
     error: Annotated[int | None, Field(description="Error code")] = None
     error_type: Annotated[str | None, Field(description="Error type")] = None
     message: Annotated[str | None, Field(description="Error message")] = None
-
-
-class HealthResponse(BaseModel):
-    """Health check response model."""
-
-    status: Annotated[str, Field(description="Health status")]
-
-
-class UserRequest(BaseModel):
```

Comment:
I assume you intended to delete everything below this line

Reply:
no, I meant to remove HealthResponse - I moved the HealthResponse to the health route. It does look more cleaner once I reorganized the models into their respective modules.